.rubocop.yml

```diff
@@ -51,6 +51,7 @@ Metrics/AbcSize:
 Metrics/BlockLength:
   Exclude:
     - "spec/**/*.rb"
+    - "app/views/chapters/show.json.jbuilder"
```

Comment:
You might be able to do something like `app/views/*/*.jbuilder`

Reply:
Done.